rc/lsp.kak

```diff
@@ -452,19 +452,19 @@ Jump to the next or previous diagnostic error" %{
         current=""
         prev=""
         selection=""
-        for e in $@; do
-            if [[ $e == *"$errorCompare"* ]]; then
+        for e in "$@"; do
+            if [ "${e#*$errorCompare}" != "$e" ]; then # e contains errorCompare
```

Comment:
Just occurred to me that you can also do:

    [ -z "${e##*${errorCompare}*}" ]

which generates the same result and it's more readable, at least for me: if `e` contains the content of `errorCompare`, the parameter expansion will return an empty string; so you test if the string is empty and ta-da!

Could sound nitpicky, but this version keeps `e` from being expanded twice, compared to the original version.

Reply:
You're right, I adapted it